grafana_client/elements/datasource.py

```diff
@@ -403,6 +440,7 @@ def health_check(self, datasource: Union[DatasourceIdentifier, Dict]) -> Datasou
         expression = get_healthcheck_expression(datasource_type, datasource_dialect)
 
         start = time.time()
+        message = "Unknown error"
```

Comment:
I discovered cases where this was needed. Apparently, not all code paths have been covering this before.